modules/task/cherami/cherami.go

```diff
@@ -46,15 +46,18 @@ const (
 )
 
 var (
-	_hyperbahnMu         sync.RWMutex
-	_hyperbahnHostsFile  string
-	_cheramiClientFunc   = cherami.NewHyperbahnClient
-	_defaultClientConfig = clientConfig{
+	_hyperbahnMu                 sync.RWMutex
+	_hyperbahnHostsFile          string
+	_cheramiClientFunc                 = cherami.NewHyperbahnClient
+	_consumedMessagesRetention   int32 = 1
```

Comment:
We can convert them to int32 when we pass it to cherami, right now it doesn't even have a measurement unit in the name..